code/src/main/java/frc/robot/Robot.java

```diff
@@ -117,7 +117,14 @@ private void configureButtonBindings() {
     winchDirectionControl.whenReleased(_container.getInstance(SetClimbNormal.class));
 
     JoystickButton start = new JoystickButton(_operatorController, _config.Controller.shooterStartupButtonPort);
-    start.whenPressed(_container.getInstance(ShooterStartupCommand.class));
+    start.whenPressed(_container.getInstance(ShooterStartupCommand.class).withRPM(_config.Shooter.shooterRPM));
+
```

Comment:
give both the names of the shooter buttons and rpm variables better, more descriptive names.  use the location of the field (trench, sector, etc) in the name